### PR TITLE
Fixes #11 - Move test sites to this repo 

### LINKED ISSUE #11: Move test sites to this repo
Replicate this file in this repo https://github.com/USGS-WiM/StreamStats-Setup/blob/master/batchTester/testSites.geojson

### PR DESCRIPTION
**Pull request by amedenblik**
Closes #11

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 
+- Update ReadMe
 - Create file structure for output files
 - Import test sites
 - Perform basin delineations
 - Compute basin characteristics
 - Compare computed basin characteristics to known values
 - Save console output to text file
-- Update ReadMe
+- Save basin characteristics output to CSV files
```
Comment (harper-wavra) [inline CHANGELOG.md:21]: should add "added test sites"

**Comment by harper-wavra** [comment]
I think this should also be into dev.

**Comment by amedenblik** [comment]
Updated base branch!

**Review by harper-wavra — CHANGES_REQUESTED** [review]
Looks good, I think just update the change log and it should be good.

**Review by harper-wavra — APPROVED** [review]

**Pull request merged**